apps/ui/src/hooks/Users/useUser.tsx

```diff
@@ -27,6 +34,15 @@ interface UserProviderProps {
 const UserProvider: React.FC<UserProviderProps> = ({ children }) => {
   const [user, setUser] = useState<User | null>(null);
   const [error, setError] = useState<string | null>(null);
+  const [loading, setLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    const storedUser = localStorage.getItem('user');
```

Comment:
use SessionStorage instead

Reply:
react use has a useSessionStorage hook you can use here https://github.com/streamich/react-use/blob/master/docs/useSessionStorage.md